### ISSUE TITLE
fix comments

### ISSUE THREAD

**Issue by Samuellekpor** [chuaindia/Pokemon-WebApp#36]
- Fix the comments of the pokemon so that it displays right after the user press the comment button

**Comment by chuaindia** [comment]
Yeah, I will try that. Because unless we refresh we can not see the
comment. But that is also a check that one commenter can not press the
comment button several times in one shot which check is necessary also.
That check is more necessary for the like button also so that one user at a
time can't do more than one like.

Thanks for the communication, Samuel.

Tanusri.
